locations/radbahn.yml

```diff
@@ -0,0 +1,93 @@
+---
+
+location: radbahn
+location_nice: Radbahn Testfeld
+latitude: 52.49928
+longitude: 13.42902
+contact_nickname: Stadtfunk gGmbH
+contacts:
+  - noc@stadtfunk.net
+
+hosts:
+
+  - hostname: radbahn-core
+    role: corerouter
+    model: ubnt_usw-flex
+
+  - hostname: radbahn-n-nf
+    role: ap
+    model: zyxel_nwa55axe
+    openwrt_version: snapshot
```

Comment:
Why do we need snapshot here? 23.05-snapshot is working just fine...

Reply:
To my knowledge mt7915e meshing f'cked in 23.05, no?

Reply:
works fine for me